### PR TITLE
[release/v2.22] Update changelogs

### PR DESCRIPTION
**Pull request by pkprzekwas**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**
/kind chore

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

File: docs/changelogs/CHANGELOG-2.22.md [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+### Bugfixes
+
+- Fix a bug where KKP managed vSphere folders are enforced but shouldn't ([#11962](https://github.com/kubermatic/kubermatic/pull/11962))
+- Fix mla-monitoring-agent configuration being invalid when custom scraping configuration is provided ([#11988](https://github.com/kubermatic/kubermatic/pull/11988))
+- Fix wrong labels in cluster/project metrics when uppercase labels were used ([#11947](https://github.com/kubermatic/kubermatic/pull/11947))
+- Set proper NodePort range in Cilium config if non-default range is used ([#11963](https://github.com/kubermatic/kubermatic/pull/11963))
+
+### Updates
+
+- Update Metering to v1.0.3 with the following changes: * Add non machine-controller managed machines to `average-cluster-machines`. Note that this is based on a new metric that will be collected together in the same release, therefore information prior this update is not available.* Fixes a bug that leads to low CPU usage values* Remove redundant label quotation ([#12035](https://github.com/kubermatic/kubermatic/pull/12035))
```
Comment (embik) [inline docs/changelogs/CHANGELOG-2.22.md]: Same here as https://github.com/kubermatic/kubermatic/pull/12042#discussion_r1135067379.

**Review by embik — CHANGES_REQUESTED** [review]

File: docs/changelogs/CHANGELOG-2.21.md [hunk 2]
```diff
@@ -7,6 +7,34 @@
...
+### Misc
+
+- Add support for ca-bundle to metering cronjobs ([#12032](https://github.com/kubermatic/kubermatic/pull/12032))
+- Allow updating of the `clusterNetwork.proxyMode` via the KKP API (PATCH endpoint) ([#12011](https://github.com/kubermatic/kubermatic/pull/12011))
+
+### Updates
+
+- Update machine-controller to v1.54.5 ([#11897](https://github.com/kubermatic/kubermatic/pull/11897))
+    - Fix an issue with Flatcar nodes not joining the cluster for clusters that don't use OSM.
+- Update Operating System Manager to v1.2.1. ([#12049](https://github.com/kubermatic/kubermatic/pull/12049))
```
Comment (hdurand0710) [inline docs/changelogs/CHANGELOG-2.21.md]: same here, for 2.21, it's OSM 1.1.3 (PR 12050)

**Review by hdurand0710 — CHANGES_REQUESTED** [review]

**Pull request merged**